src/screens/WhotoFollow/index.js

```diff
@@ -345,6 +347,20 @@ const styles = StyleSheet.create({
     marginBottom: dimen.normalizeDimen(24),
     paddingHorizontal: dimen.normalizeDimen(20)
   },
+  textShowMore: {
+    fontFamily: 'Inter-Medium',
+    fontWeight: '500',
+    fontSize: normalizeFontSize(14),
+    lineHeight: normalizeFontSize(20),
+    color: COLORS.signed_primary,
+    marginLeft: dimen.normalizeDimen(16),
+    marginBottom: dimen.normalizeDimen(16),
+    marginTop: dimen.normalizeDimen(8)
+  },
+  textShowMoreBold: {
+    fontFamily: 'Inter-Bold',
+    fontWeight: '700'
+  },
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

New styles have been added for `textShowMore` and `textShowMoreBold`. Ensure that these styles are applied correctly to the respective elements.
```diff
+ textShowMore: {
+   fontFamily: 'Inter-Medium',
+   fontWeight: '500',
+   fontSize: normalizeFontSize(14),
+   lineHeight: normalizeFontSize(20),
+   color: COLORS.signed_primary,
+   marginLeft: dimen.normalizeDimen(16),
+   marginBottom: dimen.normalizeDimen(16),
+   marginTop: dimen.normalizeDimen(8)
+ },
+ textShowMoreBold: {
+   fontFamily: 'Inter-Bold',
+   fontWeight: '700'
+ },
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->